easybuild/easyconfigs/l/LS-PrePost/LS-PrePost-4.7.eb

```diff
@@ -0,0 +1,22 @@
+easyblock = "Tarball"
+
+name = 'LS-PrePost'
+version = '4.7'
+
+homepage = 'http://lstc.com/products/ls-prepost'
+description = """LS-PrePost is an advanced pre and post-processor that is delivered free with LS-DYNA."""
+
+toolchain = SYSTEM
+
+source_urls = ['http://ftp.lstc.com/anonymous/outgoing/lsprepost/%(version)s/linux64/']
```

Comment:
and we should switch to %(version_major_minor)s here

Reply:
so, having one variable "version" which woudl in this case be 4.7.8, and one variable "version_major_minor" which would be just 4.7?

The problem is that in the url they use just 4.7

Reply:
thanks for the comment, fixed